vuepress/docs/next/docs/getting-started/ent-svc.md

```diff
@@ -1,3 +1,61 @@
 ---
 sidebarDepth: 1
 ---
+
+# Auxiliary Services
+
+The following sections discuss auxiliary services and commands.
+
+## Services
+
+When the Entando Bundle CLI enables a service, a default auxiliary configuration is generated and stored as a Docker Compose YAML file in the bundle's 'svc' folder. Three services are available out of the box: `mysql`, `postgresql` and `keycloak`.
+
+- `mysql` and `postgresql` obey the same naming convention
+   - The username and password are derived from the bundle name, e.g. YOUR-BUNDLE has the username YOUR-BUNDLE-USER and the password YOUR-BUNDLE-PASSWORD
+   - A database is automatically created and given the bundle name, e.g. YOUR-BUNDLE
+
+- `keycloak` contains the Entando-related configuration
```

Comment:
this phrase is left hanging? Colon?

Entando-related configuration >> Entando configuration

Reply:
i didn't use colons because the first `keycloak` bullet provides general info but the indents provide unrelated, additional details. a colon fits for `mysql`/`postgresql` (indents expand upon first bullet) but i wanted to be consistent. 

this one bothered me. both seem wrong for different reasons. @nshaw does the revision work?

Reply:
Formatting looks okay when I view the full file